src/MongoDB/Driver/Manager.cpp

```diff
@@ -53,6 +53,7 @@ const StaticString
 	s_MongoDBDriverManager_readpreferencetags("readpreferencetags"),
 	s_MongoDBDriverManager_readPreference("readPreference"),
 	s_MongoDBDriverManager_readPreferenceTags("readPreferenceTags"),
+	s_MongoDBDriverManager_maxStalenessMS("maxStalenessMS"),
```

Comment:
> I'm happy to add the check for all lower case though.

That sounds like a good idea, considering you do it for the other options here. If we end up changing PHPC, I can do the same (i.e. allow the canonical camel-case and all-lowercase versions).